db/migrate/20210110112042_backfill_feedback_column.rb

```diff
@@ -0,0 +1,5 @@
+class BackfillFeedbackColumn < ActiveRecord::Migration[6.0]
+  def change
+    ApplicationFeedback.all.each { |feedback| feedback.update_column(feedback: feedback.other_feedback) }
```

Comment:
`update_column` takes `(name, value)`, not `(name: value)`

Reply:
So it does. My bad